### ISSUE TITLE
Cow suggests releasing new security versions although there are no changes

### ISSUE THREAD

**Issue by NightJar** [silverstripe/cow#82]
The plan I'm currently creating is suggesting:
```
  [silverstripe/content-widget               ]        └ silverstripe/content-widget (2.0.1) new tag, prior version 2.0.0
```
But when [comparing the existing tag to the current master](https://github.com/silverstripe/silverstripe-content-widget/compare/2.0.0...dd72941) one can note that they are identical.

**Comment by tractorcow** [comment]
Yes, this is because it thinks there could be changes if we are running i18n localisations.

Even if there are no changes at this point in time, it's possible that there could be future changes.

A better behaviour would be to "guess" which modules actually could have changes. I.e. if not running --no-i18n and a `.tx` folder exists, then treat that as a candidate, rather than just treating everything as a candidate.

Also the `changelog-holder` could be modified by a new changelog being generated, so those should be added to the guess list also.

It's kind of complicated... which is why I never did this. :)
